Documentation/developer-setup-linux.md

```diff
@@ -9,52 +9,67 @@ After compiling Unreal Engine, set the following environment variables in your `
 
 ```bash
 export UNREAL_ENGINE_DIR=<path_to_unreal_engine>
-export UNREAL_ENGINE_COMPILER_DIR=$UNREAL_ENGINE_DIR/Engine/Extras/ThirdPartyNotUE/SDKs/HostLinux/Linux_x64/v17_clang-10.0.1-centos7/x86_64-unknown-linux-gnu
+export UNREAL_ENGINE_COMPILER_DIR=$UNREAL_ENGINE_DIR/Engine/Extras/ThirdPartyNotUE/SDKs/HostLinux/Linux_x64/v19_clang-11.0.1-centos7/x86_64-unknown-linux-gnu
 export UNREAL_ENGINE_LIBCXX_DIR=$UNREAL_ENGINE_DIR/Engine/Source/ThirdParty/Linux/LibCxx
 ```
 
+> Note: `v19_clang-11.0.1-centos7` is correct for Unreal Engine v4.27 and v5.0.0. It may be different for other versions of Unreal Engine.
```

Comment:
Should we add a link to https://docs.unrealengine.com/4.27/en-US/SharingAndReleasing/Linux/GettingStarted/?